database/engine/README.md

```diff
@@ -26,31 +97,32 @@ To use the database engine, open `netdata.conf` and set `[db].mode` to `dbengine
     mode = dbengine
 ```
 
-To configure the database engine, look for the `dbengine page cache size MB` and `dbengine multihost disk space MB` settings in the
-`[db]` section of your `netdata.conf`. The Agent ignores the `[db].retention` setting when using the dbengine.
+To configure the database engine, look for the `page cache size MB` and `dbengine multihost disk space MB` settings in
```

Comment:
you reverted it to a wrong setting. The new branch made this `dbengine page cache size MB`